addons/crm_livechat/static/tests/composer_tests.js

```diff
@@ -21,7 +21,10 @@ QUnit.test("Can execute lead command", async function (assert) {
         },
     });
     await openDiscuss(channelId);
-    await insertText(".o-mail-Composer-input", "/lead great lead");
+    await insertText(".o-mail-Composer-input", "/lead");
+    await click(".o-mail-Composer-suggestion");
+    await contains(".o-mail-Composer-input", { value: "/lead " });
+    await insertText(".o-mail-Composer-input", "great lead");
```

Comment:
It's strange that this test is adapted when adding a new command, isn't it possible to keep it the same?

Reply:
The way commands are handled is changed with this PR. As, now we store the selected command in composer when user selects the command from suggestion list. So, we need to add this extra `click` here.

Reply:
I don't think this change of behavior is something that we want or that is in the spec. Please double check with @paju-odoo if necessary.

Reply:
Hi @rapr-odoo @seb-odoo 
The change is okay IMO. That's the current behavior on Discord :)